#
# State Estimator Status
#
float32 s              # abscisse on path in m
uint32  idx_s          # index of abscisse on path
float32 v              # velocity in m/s
float32 dist_to_start  # predicted distances to landmarks
float32 dist_to_finish #
